### DISCUSSION TITLE
Roblox Studio

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2032] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Can anyone make a port for roblox studio if possble

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
i do not believe there is an easy way to theme roblox studio ( i havent used it since 2018 so i dunno )
if there is, please demonstrate how

**Reply by CompeyDev** [reply]
There's a pretty simple way to autogenerate themes for Roblox Studio from VSCode themes. 

Check out [rbxtheme](https://github.com/vocksel/rbxtheme).
